client/components/product-card/index.jsx

```diff
@@ -1,51 +1,47 @@
 /**
  * External dependencies
  */
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
 /**
  * Internal dependencies
  */
 import Card from 'components/card';
+import Gridicon from 'components/gridicon';
 import ProductCardPriceGroup from './price-group';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
-class ProductCard extends Component {
-	static propTypes = {
-		billingTimeFrame: PropTypes.string,
-		currencyCode: PropTypes.string,
-		description: PropTypes.oneOfType( [ PropTypes.string, PropTypes.element ] ),
-		discountedPrice: PropTypes.oneOfType( [
-			PropTypes.number,
-			PropTypes.arrayOf( PropTypes.number ),
-		] ),
-		fullPrice: PropTypes.oneOfType( [ PropTypes.number, PropTypes.arrayOf( PropTypes.number ) ] ),
-		isPurchased: PropTypes.bool,
-		subtitle: PropTypes.oneOfType( [ PropTypes.string, PropTypes.element ] ),
-		title: PropTypes.oneOfType( [ PropTypes.string, PropTypes.element ] ),
-	};
+const ProductCard = props => {
+	const {
```

Comment:
Let's list these specifically in the argument, instead of importing the entire `props` and then destructuring it.

Reply:
This has been addressed in the [latest commit](https://github.com/Automattic/wp-calypso/pull/36834/commits/c3fb1db60f042e857140c33eb869bc7ac00e84a9).